src/types/block.ts

```diff
@@ -191,7 +190,7 @@ export interface BlockTransaction {
   /** App ID when an app is created by the transaction */
   apid?: number
   /** Asset closing amount in decimal units */
-  aca?: number | bigint
+  aca?: number
```

Comment:
is this definitely number rather than bigint? I would have thought this field would be a bigint